extract-files.sh

```diff
@@ -15,9 +15,12 @@ mkdir -p $BASE/$DIR
 adb pull /system/$FILE $BASE/$FILE
 done
 
-# some extra stuff
+# Pull extra stuff from FP OS
 adb pull /system/lib/hw/audio_policy.default.so $BASE/lib/hw/audio_policy.mt6589.so
-adb pull /system/lib/libaudio.primary.default.so $BASE/lib/hw/audio.primary.mt6589.so
+adb pull /system/lib/hw/libaudio.primary.default.so $BASE/lib/hw/audio.primary.mt6589.so
```

Comment:
You said earlier:

shell@android:/system/lib/hw # ls -al
-rw-r--r-- root root 5348 2008-08-01 14:00 audio.primary.default.so
-rw-r--r-- root root 136952 2008-08-01 14:00 audio_policy.default.so
-rw-r--r-- root root 5292 2008-08-01 14:00 audio_policy.stub.so
and more files

so these files aren't in /system/lib/hw ?